src/pages/login/login.scss

```diff
@@ -25,23 +25,42 @@ page-login {
 
     .labels {
         color: rgb(96,96,96); 
-        font-size: 25px;
     }
     
+    #email:before {
```

Comment:
Nice, you got it working, can you explain to the rest of the class how this works and what you used?